environs/manual/sshprovisioner/sshprovisioner.go

```diff
@@ -185,7 +185,13 @@ func checkProvisioned(host string) (bool, error) {
 // detect the OS series and hardware characteristics.
 const detectionScript = `#!/bin/bash
 set -e
-lsb_release -cs
+os_id=$(grep '^ID=' /etc/os-release | tr -d '"' | cut -d= -f2)
```

Comment:
It will fail if /etc/os-release isn't there. AFAIK, /etc/os-release is there on every version of Ubuntu we support. We use it in https://github.com/juju/utils/blob/master/os/os_linux.go#L24 already.